client/components/site-title/index.jsx

```diff
@@ -45,17 +45,19 @@ class SiteTitleControl extends React.Component {
 				<FormFieldset>
 					<FormLabel htmlFor="blogname">{ this.props.translate( 'Site Title' ) }</FormLabel>
 					<FormTextInput
+						autoFocus
 						name="blogname"
-						value={ this.props.blogname }
 						onChange={ this.onChangeSiteTitle }
+						required
+						value={ this.props.blogname }
 					/>
 				</FormFieldset>
 				<FormFieldset>
 					<FormLabel htmlFor="blogdescription">{ this.props.translate( 'Tagline' ) }</FormLabel>
 					<FormTextInput
```

Comment:
Also, if we're looking for feature parity, we need to make this one a textarea.

Reply:
I'm inclined to stick with `FormTextInput` -- it's really called Tag*line*, and it's just a one-line text input in wp-admin, too.

Reply:
Makes sense to me, but let's also ask @MichaelArestad for confirmation, because this deviates from our mockups.